client/my-sites/plans-grid/hooks/npm-ready/data-store/is-current-plan.ts

```diff
@@ -0,0 +1,8 @@
+import { PLAN_FREE, isFreePlan, type PlanSlug } from '@automattic/calypso-products';
+
+export function isCurrentPlan( currentSitePlanSlug: string, planSlug: PlanSlug ): boolean {
```

Comment:
Something about this signature though doesn't seem correct to me. Maybe renaming to `isSamePlan` might be more precise, as that's what it does technically?

Reply:
Oh that's interesting 🤔 This wasn't even on my radar, but yeah I agree. 

Out of curiosity, how did you sense that it seemed off? Was it the redundant usage of the term "current"?

Reply:
[dbb0dea](https://github.com/Automattic/wp-calypso/pull/82786/commits/dbb0deaf7576e59cda62a67d14aa2d8fbf00b1b0)

Reply:
https://github.com/Automattic/wp-calypso/pull/82786/commits/dbb0deaf7576e59cda62a67d14aa2d8fbf00b1b0

Reply:
Not sure. I think so. It seemed a little off, hard to describe in words maybe? Not saying the alternative is better btw. It's all a bit convoluted and relative I think. Maybe the original might make more sense with a good description 🤷